### DISCUSSION TITLE
Record audio continuously

### DISCUSSION THREAD

**Discussion by ghost** [roleoroleo/yi-hack-Allwinner#363] [Q&A]
BFUS, y20ga, firmware 0.2.7
I'd like to have the cameras record audio continuously without interfering with motion detected video recordings. What would be the easiest way to accomplish this?

**Comment by roleoroleo** [comment]
With the current firmware it's not possible.
Only using an external rtsp recorder.

**Comment by ghost** [comment]
OK. Thank you for the reply! RTSP will have to do for now.

Since I'll only be using RTSP for audio recording, I'm choosing the low res video option. Which audio should I select (pcm/alaw/ulaw)? pcm works but haven't tried the others yet.

**Reply by roleoroleo** [reply]
You can use the stream without audio: ch0_2.h264
pcm/alaw/ulaw... it depends on the quality.
pcm is the best quality.
alaw and ulaw have the same quality.
